app/serializers/sprangular/payment_source_serializer.rb

```diff
@@ -10,5 +10,35 @@ def include_gateway_payment_profile_id?
     def include_gateway_customer_profile_id?
       scope.has_spree_role?("admin")
     end
+
+    def month
+      object.respond_to?(:month) ? object.month : ''
+    end
+
+    def year
+      object.respond_to?(:year) ? object.year : ''
+    end
+
+    def cc_type
+      object.respond_to?(:cc_type) ? object.cc_type : ''
+    end
+
+    def last_digits
+      object.respond_to?(:last_digits) ? object.last_digits : ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.